deploy/schema.yaml

```diff
@@ -85,3 +85,7 @@ schema: |-
     permission disable = workspace->notifications_integration_disable
     permission enable = workspace->notifications_integration_enable
   }
+
+  definition hbi/rhel_host {
```

Comment:
We are adopting the resource type name for the schema `rhe_host`

Should this change go over to https://github.com/RedHatInsights/kessel-config/blob/master/schema.zed instead ?

Reply:
it should, @alechenninger is working on that

Reply:
There is a default workspace.  It should not be required.  We are going in circles here due to the lack of a design which is reviewed and approved.  This PR should be blocked on that design.  @jewzaam will provide this design